libs/partners/anthropic/langchain_anthropic/chat_models.py

```diff
@@ -1114,27 +1060,68 @@ def _make_message_chunk_from_anthropic_event(
     event: anthropic.types.RawMessageStreamEvent,
     *,
     stream_usage: bool = True,
+    coerce_content_to_string: bool = False,
 ) -> Optional[AIMessageChunk]:
     """Convert Anthropic event to AIMessageChunk.
 
     Note that not all events will result in a message chunk. In these cases
     we return None.
     """
     message_chunk: Optional[AIMessageChunk] = None
+    # See https://github.com/anthropics/anthropic-sdk-python/blob/main/src/anthropic/lib/streaming/_messages.py  # noqa: E501
     if event.type == "message_start" and stream_usage:
         input_tokens = event.message.usage.input_tokens
         message_chunk = AIMessageChunk(
-            content="",
+            content="" if coerce_content_to_string else [],
```

Comment:
To be clear I'm good with this, but could this be a breaking change if someone was relying on content as a str

Reply:
This wouldn't be getting hit if `coerce_content_to_string` were true, which it is unless you're passing tools, which in the present version aren't streamed. But it's unnecessary, I'll remove

Reply:
Never mind, we need this since it's the first chunk. Otherwise we have to update the core logic again to replace an empty string chunk which seems messy.